products/google-cloud-sql-proxy.md

```diff
@@ -0,0 +1,37 @@
+---
+# Name of the product (mandatory).
+title: Cloud SQL Auth Proxy
+category: service
+tags: database google server-app service
```

Comment:
```suggestion
category: app
tags: google
```

Based on what I read cloud-sql-proxy is a tool that may be used on servers, but also for local development. So `app` is a better fit than `server-app`.

cloud-sql-proxy is neither a `service` or a `database`, so removed them.